api/v1alpha1/types.go

```diff
@@ -135,6 +135,9 @@ type ImageBasedUpgradeStatus struct {
 	Conditions []metav1.Condition `json:"conditions,omitempty"`
 	// +operator-sdk:csv:customresourcedefinitions:type=status,displayName="Valid Next Stage"
 	ValidNextStages []ImageBasedUpgradeStage `json:"validNextStages,omitempty"`
+	// RollbackAvailabililtyExpiration reflects the point at which rolling back may require manual recovery from expired control plane certficates.
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:text"}
+	RollbackAvailabililtyExpiration string `json:"rollbackAvailabililtyExpiration,omitempty"`
```

Comment:
why isn't this metav1.time?

Reply:
Thanks @jc-rh ! Updated to use metav1.Time field